src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/EnclaveDelegate.Crypto.cs

```diff
@@ -220,8 +218,8 @@ private string ConvertAttestationProtocolToString(SqlConnectionAttestationProtoc
                     return "HGS";
 
 #if ENCLAVE_SIMULATOR
-                case SqlConnectionAttestationProtocol.SIM:
-                    return "SIM";
+                case SqlConnectionAttestationProtocol.None:
+                    return "None";
```

Comment:
NIT: All values in switch case can be returned using enum.
```suggestion
                    return SqlConnectionAttestationProtocol.None.ToString("G");
```

Reply:
Thanks, I found a minor bug. I realized we didn't need the #if directive in that method anymore. Because of that, we can just replace the call to `ConvertAttestationProtocolToString()` with `attestationProtocol.ToString()`. Can you check if it makes sense?

Reply:
Yes, that make sense.